scripts/start.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# 'ipfs' or 'http'
+MODE=$1
+echo Running app in MODE: \"$MODE\"
+
+# Exit script as soon as a command fails.
+set -o errexit
+
+startDevchain() {
+  echo Starting devchain
+  npx aragon devchain --verbose > /dev/null &
```

Comment:
This process is not killed when the main process is, and does not check if a devchain is already running before trying to start it. I would expect a behavior similar to aragon-cli, and I think we could use their code.